### ISSUE TITLE
Vagrant prompts for admin credentials

### ISSUE THREAD

**Issue by rcw5** [geerlingguy/macbook-pro-battery-test#4]
## Issue: 

When I run `battery-test.sh` as a regular user (non-admin), vagrant is prompting me for admin credentials during the destroy process, which blocks the tests:

```
==> drupalvm: Forcing shutdown of VM...
==> drupalvm: Destroying VM and associated drives...
==> drupalvm: Pruning invalid NFS exports. Administrator privileges will be required...
Password:
```

Same thing also happens during `vagrant up`:

```
$ vagrant up
Bringing machine 'drupalvm' up with 'virtualbox' provider...
==> drupalvm: Importing base box 'geerlingguy/ubuntu1604'...
==> drupalvm: Matching MAC address for NAT networking...
==> drupalvm: Checking if box 'geerlingguy/ubuntu1604' is up to date...
==> drupalvm: Setting the name of the VM: drupalvm.dev
==> drupalvm: Pruning invalid NFS exports. Administrator privileges will be required...
Password:
```

The impact of this is that the loop fails to complete and it doesn't properly test battery life.

## Versions:
```
$ vagrant -v
Vagrant 1.9.1
```

Virtualbox 5.1.12 r112440

**Comment by geerlingguy** [comment]
Oops! I forgot that most people don't have the same sudoers config I do. Quick fix is to change sudoers like I mention in this blog post: http://www.jeffgeerling.com/blogs/jeff-geerling/better-vagrant-development-workflow

Better fix will be me setting the share type to native instead of NFS inside the Drupal VM `config.yml` file.

**Comment by geerlingguy** [comment]
Fixed by PR #5
